### PR TITLE
chore: improve README.md standardization

### PR DESCRIPTION
**Pull request by smcclellan**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Thank you for considering contributing to this product! We welcome any contributions, whether it's bug fixes, new features, or improvements to the existing codebase.
+
+### Your First Pull Request
+
+Working on your first Pull Request? You can learn how from this free video series:
+
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+
+To help you get familiar with our contribution process, we have a list of [good first issues](../../issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) that contain bugs that have a relatively limited scope. This is a great place to get started.
```
Comment (hugocaillard) [inline .github/CONTRIBUTING.md:11]:
```suggestion
To help you get familiar with our contribution process, we have a list of [good first issues](../../issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) that contain enhancements or bugs that have a relatively limited scope. This is a great place to get started.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+Transaction ID 
+Address 
+Block#  
+Time stamp
```
Comment (hugocaillard) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
```
Not applicable to clarinet I think

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (hugocaillard) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]:
```suggestion
```

No applicable

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots or consol.log to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (hugocaillard) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]:
```suggestion
**Environment (please complete the following information):**
 - OS (version): [e.g. macOS (13.4.1)]
 - Docker version [e.g. 24.0.4]
 - Clarinet version [e.g. 1.7.0] - Please use the latest version
```

**Review by hugocaillard — CHANGES_REQUESTED** [review]
Few change to adapt the template to clarinet

**Comment by smcclellan** [comment]
Thank you @hugocaillard!

**Review by hugocaillard — APPROVED** [review]

**Pull request merged**